README.rst

```diff
@@ -196,7 +196,7 @@ Related projects
 * `gmtmex <https://github.com/GenericMappingTools/gmtmex>`__: A Matlab/Octave wrapper
```

Comment:
What do you think about including the GMT repository in the list of related projects?

Reply:
Nah, I think having the link in one place at the top should be enough.